### ISSUE TITLE
Offline expired deal doesn't show as expired, only after DataCID task

### ISSUE THREAD

**Issue by RobQuistNL** [filecoin-project/boost#1279]
### Checklist

- [X] This is **not** a question or a support request. If you have any boost related questions, please ask in the [discussion forum](https://github.com/filecoin-project/boost/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/boost/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/boost/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/boost/issues) and the [discussion forum](https://github.com/filecoin-project/boost/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/boost/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [ ] I did not make any code changes to boost.

### Boost component

- [X] boost daemon - storage providers
- [X] boost client
- [ ] boost UI
- [X] boost data-transfer
- [ ] boost index-provider
- [ ] Other

### Boost Version

```text
1.6.0
```

### Describe the Bug

I had an older offline deal, which I tried to import. Only after doing the entire DataCID calculdation, the deal was marked as expired.

### Logging Information

```text
Deal Logs
2023-02-27 17:37:19.096		
executing deal proposal received from network
peer: 12xxxxx
2023-02-27 17:37:20.275	1s	
processing deal acceptance request
2023-02-27 17:37:21.196	921ms	
offline deal accepted, waiting for data import
2023-02-28 16:20:14.606	a day	
restarted deal: waiting for offline deal data import
2023-03-01 06:55:34.997	15h	
restarted deal: waiting for offline deal data import
2023-03-14 15:35:08.197	13 days	
restarted deal: waiting for offline deal data import
2023-03-14 15:54:47.387	20m	
restarted deal: waiting for offline deal data import
2023-03-14 16:04:33.273	10m	
restarted deal: waiting for offline deal data import
2023-03-14 17:35:18.267	2h	
restarted deal: waiting for offline deal data import
2023-03-14 19:35:21.263	2h	
restarted deal: waiting for offline deal data import
2023-03-15 02:59:13.183	7h	
restarted deal: waiting for offline deal data import
2023-03-15 03:37:01.352	38m	
import data for offline deal
filepath: /mnt/xxxxxkbi.car
2023-03-15 03:37:01.353	1ms	
processing deal acceptance request
2023-03-15 03:37:01.356	3ms	
tagged funds for deal
tagged for deal collateral: 12,251,340,281,427,465
tagged for deal publish: 150,000,000,000,000,000
total available for collateral: 49,873,906,641,611,325,145
total available for publish: 4,109,586,894,769,661,628,892
total tagged for collateral: 1,163,893,503,036,244,414
total tagged for publish: 14,250,000,000,000,000,000
2023-03-15 03:37:01.356	0ms	
offline deal data imported and deal scheduled for execution
2023-03-15 03:37:01.356	0ms	
deal execution initiated
deal state: 
2023-03-15 03:37:01.357	1ms	
size of imported offline deal file
filepath: /mnt/xxxxxbi.car
size: 20,226,565,172
2023-03-15 03:37:01.358	1ms	
deal execution in progress
2023-03-15 03:37:01.359	1ms	
checking commP
2023-03-15 03:37:56.906	1m	
commp matched successfully for imported data for offline deal
2023-03-15 03:37:56.907	1ms	
updated deal checkpoint in DB
new checkpoint: Transferred
old checkpoint: Accepted
2023-03-15 03:37:56.908	1ms	
deal failed
err: deal proposal must be proven on chain by deal proposal start epoch 2660474, but it has expired: current chain height: 2684715
2023-03-15 03:37:56.909	1ms	
cleaning up deal
2023-03-15 03:37:56.910	1ms	
deal finished
2023-03-15 03:37:56.911	1ms	
untagged funds for deal as deal finished
err: 
untagged collateral: 12,251,340,281,427,465
untagged publish: 150,000,000,000,000,000
2023-03-15 03:37:56.912	1ms	
finished cleaning up deal
2023-03-15 03:37:56.912	0ms	
deal go-routine finished execution
```

### Repo Steps

1. Make offline deal
2. let it expire
3. see nothing changes in deal UI
4. import it
5. see it break with `Error: deal proposal must be proven on chain by deal proposal start epoch 2660474, but it has expired: current chain height: 2684715`

**Comment by LexLuthr** [comment]
Closing as a duplicate of https://github.com/filecoin-project/boost/issues/1277

**Comment by RobQuistNL** [comment]
:facepalm: sorry :sweat_smile:
